src/ai/backend/manager/models/alembic/versions/1d42c726d8a3_add_container_registry_tables.py

```diff
@@ -0,0 +1,241 @@
+"""Migrate container registry schema from `etcd` to `postgreSQL`
+
+Revision ID: 1d42c726d8a3
+Revises: 75ea2b136830
+Create Date: 2024-03-05 10:36:24.197922
+
+"""
+
+import asyncio
+import json
+import os
+import sys
+from concurrent.futures import ThreadPoolExecutor
+from itertools import groupby
+from queue import Queue
+from typing import Any, Final, Mapping, cast
+
+import sqlalchemy as sa
+import trafaret as t
+from alembic import op
+
+from ai.backend.common import validators as tx
+from ai.backend.common.etcd import AsyncEtcd, ConfigScopes
+from ai.backend.manager.config import load
+from ai.backend.manager.models.base import IDColumn, StrEnumType, convention
+from ai.backend.manager.models.container_registry import ContainerRegistryRow, ContainerRegistryType
```

Comment:
```suggestion
from ai.backend.manager.models.container_registry import ContainerRegistryType
```

It is highly recommended to define a table in migration files rather than to import a table because `ContainerRegistryRow` can change but DB schemas could be remain outdated.
It would be also good to define enum types in migration files.